### ISSUE TITLE
[0.2.0] Tests are failing without logs and harness contains only error code

### ISSUE THREAD

**Issue by StykMartin** [beaker-project/restraint#26]
```
** Preparing metadata
** Refreshing peer role hostnames: Retries 0
** Updating env vars
** Updating external watchdog: 5400 seconds
** Installing dependencies
** Running task: 108787822 [/distribution/containers/test-scl-exec]
Skipping Multihost sync .. SERVERS/CLIENTS roles not set
** ERROR: Command returned non-zero 256
** Completed Task : 108787822
```
No task output included.
This makes me feel that plugins running at the beginning of execution blows up.

You can find additional information on job `J:4187178`.

**Comment by lukaszachy** [comment]
FTR: Running ```J:4187719``` with restraint-rhts-0.1.45 and so far no issue happened.

**Comment by cbouchar** [comment]
Need more details.  Job 4187178 seems to have task output.

**Comment by StykMartin** [comment]
- T:108787821 
- T:108787822
- T:108787826 
and so on

**Comment by danrodrig** [comment]
This is fixed in #50. Running build from PR in the internal Beaker production instance, `J:4225987`, behaves as 0.1.45.
